src/core/Controller.cpp

```diff
@@ -220,24 +220,11 @@ Controller * Controller::create( ControllerType _ct, Model * _parent )
 
 Controller * Controller::create( const QDomElement & _this, Model * _parent )
 {
-	Controller * c;
-	if( static_cast<ControllerType>(_this.attribute( "type" ).toInt()) == ControllerType::Peak )
-	{
-		c = PeakController::getControllerBySetting( _this );
-	}
-	else
-	{
-		c = create(
-			static_cast<ControllerType>( _this.attribute( "type" ).toInt() ),
-										_parent );
-	}
-
-	if( c != nullptr )
-	{
-		c->restoreState( _this );
-	}
-
-	return( c );
+	const auto controllerType = static_cast<ControllerType>(_this.attribute("type").toInt());
+	auto controller = controllerType == ControllerType::Peak ? PeakController::getControllerBySetting(_this)
+															 : create(controllerType, _parent);
```

Comment:
Depending on editor tab width, this can get pretty far out there. Let's keep it a single tab. 😁 
```suggestion
	auto controller = controllerType == ControllerType::Peak
		? PeakController::getControllerBySetting(_this)
		: create(controllerType, _parent);
```